### ISSUE TITLE
Log Viewer Corruption

### ISSUE THREAD

**Issue by Amoeba00** [Maassoft/ColorControl#239]
1 - Love the new view of the log viewer.  Very nice and so much easier to read.

2 - This happened in the other versions as well, but just now reporting it.  When selecting the Log Level, the drop down menu isn't really selectable - as it seems to disappear - and  you can't really select anything - so it stays in debug mode.  Tried with mouse and just keyboard - both the same results.  Hope that makes sense.

![image](https://github.com/Maassoft/ColorControl/assets/39176113/afcf847d-d7db-43d3-ba51-778dee57f5f6)

**Comment by Maassoft** [comment]
Weird, that's not happening on my system.
If I search for this problem this seemed to be a WPF bug with transparency, but a long time ago.
Actually, what happened in the other versions? Do you mean the drop down in the Novideo sRGB advanced window?

**Comment by Amoeba00** [comment]
Yes! Behaved the same way - and still does in this version.  I never mucked around with those settings so didn't want to add to your plate if it was just isolated to my system - but now it's affecting something I check.

**Comment by Amoeba00** [comment]
Just wanted to provide update - this still happens on 10.3.  No clue what's going on - and I've never had a need to get into the logs - but basically - the only other log levels I can select is Trace and Info - and Info is weird because I have to move the mouse slowly for it to appear and then select it.  Unable to select Warning, Error, or Fatal.

<img width="1029" height="392" alt="Image" src="https://github.com/user-attachments/assets/0dc4051c-9a74-4735-a740-106a24f722dd" />
